packages/grafana-ui/src/components/Slider/RangeSlider.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FunctionComponent } from 'react';
+import { Range as RangeComponent, createSliderWithTooltip } from 'rc-slider';
+import { cx } from 'emotion';
+import { Global } from '@emotion/core';
+import { useTheme } from '../../themes/ThemeContext';
+import { getStyles } from './styles';
+import { RangeSliderProps } from './types';
+
+/**
+ * @public
+ *
+ * RichHistoryQueriesTab uses this Range Component
```

Comment:
Think we can skip this comment. We should tend to add comments that are meaningful for someone consuming grafana/ui APIs. If I were developing a new plugin and wanted to use this component I would find this information unnecessary.